kusto/ingest/internal/resources/resources.go

```diff
@@ -175,14 +178,14 @@ func (m *Manager) Close() {
 
 func (m *Manager) renewResources() {
 	tickDuration := 30 * time.Second
-	tickTotal := 1 * time.Hour
+
 	tick := time.NewTicker(tickDuration)
 	count := 0 * time.Second
 	for {
 		select {
 		case <-tick.C:
 			count += tickDuration
-			if count >= tickTotal {
+			if count >= fetchInterval {
```

Comment:
Consider if you want fetch retry to loop for ever if the service is down for example

Reply:
Not sure I understand how this comment is different from the last one.

Reply:
Fixed.